content/creating-commands.md

```diff
@@ -35,13 +35,18 @@ A command can be configured the following way:
 - `args`: list of arguments to always prepend to the command
 - `timeout`: how long we allow the command to run until we cancel it, in
   seconds, 60 by default
-- `auth_strategy`: defined the authorization strategy
+- `auth_strategy`: defines the authorization strategy
   - `any`: everyone will be allowed to run this command
   - `none`: no user will be allowed to run this command (default value,
     permissions have to be explicit and conscious)
   - `group`: use `allowed_groups` to control who has access to this command
 - `allowed_groups`: list of groups allowed to run this command
+- `channel_strategy`: defines the channel authorization strategy
+  - `any`: the command can be invoked from any channel
+  - `im_only`: the command can only be invoked in an IM conversation
```

Comment:
Isn't it DM? In any case, acronyms. 😡

Reply:
I can change it.

Not sure about making it too large, but in any case, what's the specific proposal?

Reply:
>the command can only be invoked in an direct message conversation

or
>the command can only be invoked in an private conversation

Something more explicit than an acronym.

Reply:
gotcha, now I understand what you were talking about.

Will do.

Reply:
BTW, IM comes from the Slack API itself: https://github.com/nlopes/slack/blob/db68538b374a5f052bf6befea117ac74760f4e8e/websocket_dm.go#L4